src/components/application_manager/src/commands/mobile/subscribe_button_request.cc

```diff
@@ -93,7 +93,8 @@ void SubscribeButtonRequest::Run() {
 bool SubscribeButtonRequest::IsSubscriptionAllowed(
     ApplicationSharedPtr app, mobile_apis::ButtonName::eType btn_id) {
   if (!app->is_media_application() &&
-      ((mobile_apis::ButtonName::SEEKLEFT == btn_id) ||
```

Comment:
@LuxoftAKutsan Created PRs for ATF tests: smartdevicelink/sdl_atf#49, smartdevicelink/sdl_atf_test_scripts#19